src/main/sensors/voltage.c

```diff
@@ -243,6 +253,7 @@ void voltageMeterESCReadMotor(uint8_t motorNumber, voltageMeter_t *voltageMeter)
     if (escData) {
         voltageMeter->unfiltered = escData->dataAge <= ESC_BATTERY_AGE_MAX ? escData->voltage : 0;
         voltageMeter->filtered = voltageMeter->unfiltered; // no filtering for ESC motors currently.
+        voltageMeter->sagFiltered = voltageMeter->unfiltered; // no filtering for ESC motors currently. TODO why is this?
```

Comment:
Answer: Because they are already filtered on the ESC side. Plus the update frequency for them is < 50 Hz, so the filter will be working on stale data most of the time anyway.